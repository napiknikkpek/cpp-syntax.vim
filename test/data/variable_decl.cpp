int a;
int const a;
int* a;
int const* a;
int* const a;
int const* const a;
int const& a;
int& const a;
int const& const a;
A a;
A const a;
A* a;
A const* a;
A* const a;
A const* const a;
A const& a;
A& const a;
A const& const a;
auto a;
auto& a;
auto const& a;
auto& const a;
auto const& const a;
auto* a;
auto const* a;
auto* const a;
auto const* const a;
a * b;
return a * b;
a & b;
return a & b;
A<X> a;
A<X> const a;
A<X>* a;
A<X>const* a;
A<X>* const a;
A<X> const* const a;
A<X>& a;
A<X>const& a;
A<X>& const a;
A<X> const& const a;
const A a;
const auto a;
A&& a;
auto&& a;
A<X>&& a;
