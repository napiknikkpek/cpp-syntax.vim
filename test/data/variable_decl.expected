zzz.x.
zzz.aaaaa.x.
zzz..x.
zzz.aaaaa..x.
zzz..aaaaa.x.
zzz.aaaaa..aaaaa.x.
zzz.aaaaa..x.
zzz..aaaaa.x.
zzz.aaaaa..aaaaa.x.
t.x.
t.aaaaa.x.
t..x.
t.aaaaa..x.
t..aaaaa.x.
t.aaaaa..aaaaa.x.
t.aaaaa..x.
t..aaaaa.x.
t.aaaaa..aaaaa.x.
aaaa.x.
aaaa..x.
aaaa.aaaaa..x.
aaaa..aaaaa.x.
aaaa.aaaaa..aaaaa.x.
aaaa..x.
aaaa.aaaaa..x.
aaaa..aaaaa.x.
aaaa.aaaaa..aaaaa.x.
x...x.
aaaaaa.x...x.
x...x.
aaaaaa.x...x.
t.t..x.
t.t..aaaaa.x.
t.t...x.
t.t.aaaaa..x.
t.t...aaaaa.x.
t.t..aaaaa..aaaaa.x.
t.t...x.
t.t.aaaaa..x.
t.t...aaaaa.x.
t.t..aaaaa..aaaaa.x.
aaaaa.t.x.
aaaaa.aaaa.x.
t...x.
aaaa...x.
t.t....x.
